test/inlining/rlexe.xml

```diff
@@ -274,4 +274,10 @@
       <compile-flags>-loopinterpretcount:1 -bgjit- -force:inline</compile-flags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>bug9936017.js</files>
+      <compile-flags>-off:simplejit -oopjit-</compile-flags>
```

Comment:
so, the default compile flags that the interpreted (misnomer) flavor of our unittests get run with are -maxinterpretcount:1 and -maxsimplejitruncount:1. While -off:simplejit and -oopjit- are good to use for focussed private testing, as much as possible, we should try to checkin tests which can test the intended behaviour with -maxinterpretcount:1 and -maxsimplejitruncount:1. For example, in your case, having a third call to test1 and running with -mic:1 -msjrc:1 would also repro the issue.